requirements.txt

```diff
@@ -1,5 +1,7 @@
+config
 flask
 sqlalchemy
 flask-sqlalchemy
 flask-socketio
 pyscard
+psycopg2
```

Comment:
On my machine this downloaded source code and tried to compile. Wheel is used for `psycopg2-binary`.